### ISSUE TITLE
the bounding box isn't sufficiently large to cover poses!

### ISSUE THREAD

**Issue by LiBinbinBUPT** [AlvinYH/Faster-VoxelPose#5]
Dear author! When I try to debug such problem,it seems that the problem comes from the pytorch dataloader! How to solve such problem?

**Comment by AlvinYH** [comment]
Actually, this printed message about the bounding box doesn't suggest the occurrence of errors. It's simply a **warning** that the tight bounding box isn't large enough to cover poses in a few cases. You can increase the size of the bounding box. However, the growing size of voxels will introduce larger quantization errors. It's a trade-off between dealing with some corner cases and maintaining accuracy.

**Comment by LiBinbinBUPT** [comment]
> 

All right, thanks for your answer, my machine seems to operate slowly, maybe my machine's congures is too old（RTX2060 6G and i7 8700）.

**Comment by FrankMMMMMM** [comment]
> > 
> 
> All right, thanks for your answer sincerely, my machine seems to operate slowly, maybe my machine is too old（RTX2060 6G and i7 8700）.

Hello. Can this problem be solved by upgrading the hardware device?
